qdrant_client/async_qdrant_remote.py

```diff
@@ -112,10 +113,14 @@ def __init__(
             self._rest_headers["api-key"] = api_key
             self._grpc_headers.append(("api-key", api_key))
         grpc_compression = kwargs.pop("grpc_compression", None)
```

Comment:
could you add Optional[grpc.Compression] please?

Reply:
should it be like this?
```python
grpc_compression: Optional[grpc.Compression] = kwargs.pop("grpc_compression", None)
```

Reply:
yeap